src/components/Header/AboutModal.tsx

```
@@ -0,0 +1,103 @@
+import { ExternalLink } from '~/shared';
+
+
+export interface AboutModalProps {
+  isOpen: boolean;
+  onClose: () => void;
+}
+
+const AboutModal: React.FC<AboutModalProps> = ({ isOpen, onClose }) => {
+  const GITHUB_REPOSITORY_URL="https://github.com/konflux-ci/konflux-ui"
+  const OFFICIAL_WEBSITE_URL=""; // Not yet implemented
+  const [parsedData] = useKonfluxPublicInfo();
```

ESLint

[error] 'parsedData' is assigned a value but never used. Allowed unused vars must match /React/u.

(@typescript-eslint/no-unused-vars)